packages/cli/src/commands/worker.ts

```diff
@@ -193,7 +199,7 @@ export class Worker extends BaseCommand {
 				);
 				await additionalData.hooks.executeHookFunctions('workflowExecuteAfter', [failedExecution]);
 			}
-			return { success: true };
+			return { success: true, error: error as ExecutionError };
```

Comment:
We can also remove this property since this is not in use at the moment

Reply:
that one was actually being used but i just noticed that merging master in removed it...